src/main/java/kubatech/tileentity/gregtech/multiblock/GT_MetaTileEntity_ExtremeIndustrialGreenhouse.java

```diff
@@ -826,16 +823,20 @@ protected Widget createCropsContainerWidget(EntityPlayer player) {
                         return;
                     }
                 }
-                if (clickData.mouseButton == 1) {
-                    if (player.inventory.getItemStack() == null) {
-                        player.inventory.setItemStack(removed.input);
-                        ((EntityPlayerMP) player).isChangingQuantityOnly = false;
-                        ((EntityPlayerMP) player).updateHeldItem();
+                if (clickData.ctrl) {
+                    if (!addOutput(removed.input)) {
+                        mStorage.add(removed);
```

Comment:
Add back where it was taken
```suggestion
                        mStorage.add(realID, removed);
```